chain/client/src/sync/state.rs

```diff
@@ -247,6 +277,7 @@ impl StateSync {
                             }
                         }
                     }
+                    // If all parts are done - we can move towards scheduling (TODO: why?)
```

Comment:
This TODO is redundant given another TODO at line 293.

Reply:
good point. done.